README.md

```diff
@@ -1,2 +1,7 @@
 # identity-serializer-fst
-We have a requirement to have a fast serializer on behalf of Java serializer for performance improvements
+Fast serialization implementation that can be used on behalf of default java serializer
+## Instructions to integrate with wso2 identity server
+
+- Add the build jar file to <IS-HOME>/repository/components/dropins
+- Download javassist [3.21.0-GA](https://repo1.maven.org/maven2/org/javassist/javassist/3.21.0-GA/javassist-3.21.0-GA.jar) and place in <IS-HOME>/repository/components/lib
```

Comment:
Add the reason as below
"Javassist is a library to instrument the bytecode. This library need to be on parent class loader than the OSGI framework classloader. Hence it need to be placed in common lib folder"